modules/__tests__/Router-test.js

```diff
@@ -272,7 +268,7 @@ describe('Router', function () {
         render() {
           const { label, children } = this.props
           const child = React.cloneElement(children, {
-            createElement: this.createElement
+            createElement: this.createElement.bind(this)
```

Comment:
Yes - it breaks referential equality, which leads to e.g. pure render not working. It seldom ever really matters, but it's just good hygiene.